stylus/ui-app/alerts.styl

```diff
@@ -21,23 +31,46 @@ $alerts
         background-color emerald
         box-shadow       0 em(6px) em(18px) 0 rgba(50, 54, 63, .23)
         box-sizing       border-box
-        padding          1em
+        padding          1em 1.5em
         color            white
+        opacity          1
 
-        //                  When only a title is displayed
-        min-height          em(96px)
-        padding-left        em(96px)
-        background          embedurl('../../assets/icons/icon-check.svg') no-repeat em(24px) center
-        transition                 top .5s ease-out
+        transition                 top .5s ease-out, opacity .2s ease-out
         cursor                     pointer
 
+        p
+          margin 0
+          line-height 1.5
+
         .coz-alert-title
             font-weight bold
 
     .coz-alert--error
         background-color pomegranate
-        background-image embedurl('../../assets/icons/icon-warn.svg')
 
+        .coz-btn
+          color         pomegranate
+          border        none
+          font-weight   bold
+          margin        .5rem 0 0
+          padding       .5rem 1rem
+          height        auto
+          border-radius em(3px)
+
+          &:hover
+            color       darken(pomegranate, 25%)
+
+    &.coz-alerter--hidden
+      .coz-alert
+        top                        -25%
+        opacity                    0
+        transition-timing-function ease-in
+
+      .coz-overlay
+        opacity 0
+
+    // Deprecated update existing code and remove
+    // User .coz-alerter--hidden on .coz-alerter instead
     .coz-alert--closing
```

Comment:
Well we should limit changes as much as possible, but if we have to a changelog and an update on the version should be done, obviously. That being said, we're still on the early stage of Cozy-UI. I think we can skip on this for the time being, until we have a stable Cozy-UI repo. Don't you think? /poke @m4dz